File: Step-1/CPP/Step-1.1/2.DataTypes/datatypes_problem.cpp
#include <bits/stdc++.h>
using namespace std;

int datatypes(string type){
    if (type == "int") return 4;
    else if (type == "Long") return 8;
    else if (type == "Float") return 4;
    else if (type == "Double") return 8;
    else if (type == "Character") return 1;
    else return -1;
}

int main(){
    string type;
    cin>>type;
    int ans = datatypes(type);
    cout << ans << endl;
    return 0;
}

File: Step-1/Python/Step-1.1/2.DataTypes/datatypes_problem.py
def datatypes(type):
    if type == "int":
        return 4
    elif type == "Long":
        return 8
    elif type == "Float":
        return 4
    elif type == "Double":
        return 8
    elif type == "Character":
        return 1
    else:
        return -1

def main():
    type = input()
    ans = datatypes(type)
    print(ans)

if __name__ == "__main__":
    main()